modules/install/join_rke2_master.sh

```diff
@@ -132,6 +179,7 @@ main() {
   subscription_manager
   disable_cloud_setup
   install
+  install_etcdctl
```

Comment:
We should only install etcdctl on etcd nodes. This will currently run on cp-only nodes as well if that is passed in the configuration.